cmd/btfs/functest.go

```diff
@@ -68,6 +79,15 @@ func add_functest(btfsBinaryPath string) error {
 	}
 
 	cmd := exec.Command(btfsBinaryPath, "id")
+	go func() {
```

Comment:
this is btfs id, which get from local, no need to collect network.
Do you want to do the time out here?

Reply:
per discussed in person, we don't have a solid case on how this will happen, thus we don't know the consequence.  it's placed there just as a safe guard.